pkg/conduit/config_test.go

```diff
@@ -15,13 +15,15 @@
 package conduit
 
 import (
+	"github.com/matryer/is"
```

Comment:
Imports are organized in groups, with a blank line between them. The first group is always the standard lib stuff: https://github.com/golang/go/wiki/CodeReviewComments#imports.